machines/vic/certificate.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"time"
+
+	"github.com/vmware/vic/pkg/errors"
+)
+
+type Keypair struct {
+	tlsGenerate bool
+	keyFile     string
+	certFile    string
+
+	CertPEM string
+	KeyPEM  string
+}
+
+func NewKeyPair(tlsGenerate bool, keyFile, certFile string) *Keypair {
+	return &Keypair{
+		tlsGenerate: tlsGenerate,
+		keyFile:     keyFile,
+		certFile:    certFile,
+	}
+}
+
+func (k *Keypair) generate() error {
+	org := "VMware"
```

Comment:
This should not be "VMware" - if it's possible to use an empty string, do so, otherwise something that makes it apparent that it's an installer generated self-signed cert.